docs/input/docs/installation/configure.md

```diff
@@ -0,0 +1,51 @@
+---
+Order: 20
+Title: Configuration
+Description: Configure BBT.StructureTools
+---
+
+The BBT Structure Tools are entirely configurable by code. This section covers the
+correct IoC container registration of the infrastructure components.
+
+## Preconditions
+
+* An IoC container can be used to resolve registrations as well as internally by the library.
```

Comment:
I don't understand the last part of this sentence (`...as well as internally be the library`).

Reply:
Sentence rewritten